test/versioned/bunyan/bunyan.tap.js

```diff
@@ -0,0 +1,315 @@
+      // Example Bunyan setup to test
+      const mockStream = makeSink()
+      const logger = bunyan.createLogger({
+        name: 'test-logger',
+        stream: mockStream
+      })
+
+      logStuff({ logger, helper, agent })
+
+      t.same(agent.logs.getEvents(), [], 'should not add any logs to log aggregator')
+      mockStream.logs.forEach((line) => {
+        originalMsgAssertion({
+          t,
+          includeLocalDecorating: true,
+          hostname: agent.config.getHostnameSafe(),
+          logLine: JSON.parse(line)
+        })
+      })
+      t.end()
+    })
+  })
+
+  t.test('log forwarding enabled', (t) => {
+    t.autoend()
+
+    t.beforeEach(() => {
+      setup({
+        application_logging: {
+          enabled: true,
+          forwarding: {
+            enabled: true
+          },
+          local_decorating: {
+            enabled: false
+          },
+          metrics: {
+            enabled: false
+          }
+        }
+      })
+    })
+
+    t.test('should add linking metadata to log aggregator', (t) => {
+      // Example Bunyan setup to test
+      const mockStream = makeSink()
+      const logger = bunyan.createLogger({
+        name: 'test-logger',
+        stream: mockStream
+      })
+
+      logStuff({ logger, helper, agent })
+
+      const msgs = agent.logs.getEvents()
+      t.equal(msgs.length, 2, 'should add both logs to aggregator')
+      msgs.forEach((msg) => {
+        logForwardingMsgAssertion(t, msg, agent)
+      })
+
+      mockStream.logs.forEach((logLine) => {
+        originalMsgAssertion({
+          t,
+          logLine: JSON.parse(logLine),
+          hostname: agent.config.getHostnameSafe()
+        })
+      })
+      t.end()
+    })
+
+    t.test('should properly reformat errors on msgs to log aggregator', (t) => {
+      const name = 'TestError'
+      const errorMsg = 'throw uncaught exception test'
+      // Simulate an error being thrown to trigger Winston's error handling
+      class TestError extends Error {
+        constructor(msg) {
+          super(msg)
+          this.name = name
+        }
+      }
+      const err = new TestError(errorMsg)
+
+      const mockStream = makeSink()
+      const logger = bunyan.createLogger({
+        name: 'test-logger',
+        stream: mockStream
+      })
+      logger.error({ err })
+
+      const msgs = agent.logs.getEvents()
+      t.equal(msgs.length, 1, 'should add error line to aggregator')
+      const [msg] = msgs
+      t.equal(msg['error.message'], errorMsg, 'error.message should match')
+      t.equal(msg['error.class'], name, 'error.class should match')
+      t.ok(typeof msg['error.stack'] === 'string', 'error.stack should be a string')
+      t.notOk(msg.stack, 'stack should be removed')
+      t.notOk(msg.trace, 'trace should be removed')
+      t.end()
+    })
+  })
+
+  t.test('metrics', (t) => {
+    t.autoend()
+
+    t.test('should log unknown for fatal log level', (t) => {
```

Comment:
this should log accordingly, I think we missed adding this as a constant as I see pino support this too. You can just add it to the levels constants [here](https://github.com/newrelic/node-newrelic/blob/main/lib/metrics/names.js#L276) then you can just remove this test and include in the one below.